composites/Plugin/SnippetEditor/components/SnippetEditorFields.js

```diff
@@ -199,6 +262,33 @@ class SnippetEditorFields extends React.Component {
 		}
 	}
 
+	/**
+	 * Inserts a % into a ReplacementVariableEditor to trigger the replacement variable suggestions.
+	 *
+	 * @param {string} fieldName The field name to get the ref for.
+	 *
+	 * @returns {void}
+	 */
+	triggerReplacementVariableSuggestions( fieldName ) {
+		const element = this.elements[ fieldName ];
+
+		element.triggerReplacementVariableSuggestions();
```

Comment:
I find very confusing this function has the same name of the one that comes from the ReplacementVariableEditor class. Any specific reason why they should have the same name?

Reply:
Hm no great ideas but I'd say the one that actually triggers the suggestions is the one from the ReplacementVariableEditor class? Instead, this one gets a field name an passes it to the other one.